features/cli/deploy.feature

```diff
@@ -77,7 +77,7 @@ Feature: deployment related features
       | f | deployment1.json |
     Then the step should succeed
     Given I obtain test data file "deployment/updatev1.json"
-    And I wait up to 60 seconds for the steps to pass:
+    And I wait up to 30 seconds for the steps to pass:
```

Comment:
I doubt this is the root cause of failure.
wait up to xx seconds means we wait xx seconds at most if the situation does not met (then timeout and fail the scenario). If the situation is met earlier, then we just break out the wait loop and continue execution the following steps.

Reply:
@liangxia i agree, i need to figure out what is causing this issue and fix it, i will mark this as hold for now !!

Reply:
@kasturinarra  I'm also can't figure out why hit this error , but I think we should change test logic to follow:
1) create dc;
2) check the first deploy succeed ;
3) replace the dc;
4) check the second deploy succeed;
5) rollback . 
Is this ok ?

Reply:
@zhouying7780 should we just wait for the hooks-1-deploy and its pod to be in completed state before rolling back ? Do you think that will help ?